### PR TITLE
get_lexer manual coverage

### PR DESCRIPTION
**Pull request by nedsko**

### CODE REVIEW

File: httpie/output/formatters/colors.py [hunk 1]
```diff
@@ -111,22 +116,36 @@ def get_lexer(mime, explicit_json=False, body=''):
...
     else:
+        has_branched('get_lexer', 5)
         for name in lexer_names:
             try:
                 lexer = pygments.lexers.get_lexer_by_name(name)
             except ClassNotFound:
+                has_branched('get_lexer', 6)
                 pass
 
     if explicit_json and body and (not lexer or isinstance(lexer, TextLexer)):
```
Comment (jberglinds) [inline httpie/output/formatters/colors.py:130]: This if should have a check right?

**Review by jberglinds — CHANGES_REQUESTED** [review]
Minor thing, good otherwise

**Review by robban112 — APPROVED** [review]

File: httpie/output/formatters/colors.py (hunk 1 above)
Comment (nedsko) [reply httpie/output/formatters/colors.py:130]: What sort of check are you referring too? I check if the separate conditions are true below, on line 141.

Comment (nedsko) [reply httpie/output/formatters/colors.py:130]: Nevermind now I get what you mean 😅

Comment (ghost) [reply httpie/output/formatters/colors.py:130]: The if-statement should be split up

**Review by ghost — DISMISSED** [review]
Needs to split up if-statement looks good otherwise

File: httpie/output/formatters/colors.py [hunk 2]
```diff
@@ -111,22 +116,37 @@ def get_lexer(mime, explicit_json=False, body=''):
...
+            has_branched('get_lexer', 13)
             lexer = pygments.lexers.get_lexer_by_name('json')
+    # For if-statement above
+    if explicit_json:
+        has_branched('get_lexer', 7)
+        if body:
+            has_branched('get_lexer', 8)
+            if not lexer:
+                has_branched('get_lexer', 9)
+            if isinstance(lexer, TextLexer):
```
Comment (jberglinds) [inline httpie/output/formatters/colors.py]: I think this one should be an `elif`

File: httpie/output/formatters/colors.py (hunk 1 above)
Comment (jberglinds) [reply httpie/output/formatters/colors.py:130]: It's checked below which branch was actually taken.

Comment (ghost) [reply httpie/output/formatters/colors.py:130]: I see, nvm then

File: httpie/output/formatters/colors.py (hunk 2 above)
Comment (ghost) [reply httpie/output/formatters/colors.py]: I agree, can fix it

**Review by jberglinds — APPROVED** [review]

**Pull request merged**